styles.css

```diff
@@ -113,16 +114,321 @@
+  transition: opacity 1s;
 }
 
-.main__button:hover {
-  opacity: .5;
+.main__button-blic {
+  position: relative;
+  margin: 0;
+  width: 10px;
+  height: 175px;
+  background: #fff;
+  transform: rotate(-45deg);
+  opacity: .1;
+  animation: example 5s infinite ease-in-out alternate-reverse;
+}
+
+@keyframes example {
+  0% {
+    left: 0;
+    top: -70x;
+  }
+
+  50% {
+    left: 370px;
+    top: -70px;
+  }
+
+  100% {
+    left: 0;
+    top: -70px;
+  }
+}
+
+@media (max-width: 1500px) {
+  .wrapper {
+    width: 935px;
+  }
+
+  .header {
+    height: 34px;
+    margin-bottom: 120px;
+  }
+
+  .menu {
+    display: flex;
+  }
+
+  .menu__item {
+    margin-left: 24px;
+    font: 15px/51px roboto, helvetica, arial, sans-serif;
+  }
+
+  .menu__item_active::after {
+    display: block;
+    top: 35px;
+  }
+
+  .main {
+    margin-top: 0;
+    background: top right no-repeat url(img/frame1024.svg);
+    background-size: 55%;
+  }
+
+  .main__title {
+    margin: 0;
+    font: 50px/66px pt_serifbold, arial, helvetica, sans-serif;
+  }
+
+  .main__subtitle {
+    margin-top: 40px;
+    padding: 0;
+    font: 17px/20px roboto, arial, helvetica, sans-serif;
+  }
+
+  .main__button {
+    margin-top: 41px;
+    padding: 13px 16px 15px 23px;
+    font: 14px/16px roboto, arial, helvetica, sans-serif;
+  }
+}
+
+@media (max-width: 1300px) {
+  .wrapper {
+    width: 93%;
+  }
+
+  .header {
+    height: 34px;
+    margin-bottom: 120px;
+  }
+
+  .menu {
+    display: flex;
+  }
+
+  .menu__item {
+    margin-left: 24px;
+    font: 15px/51px roboto, helvetica, arial, sans-serif;
+  }
+
+  .menu__item_active::after {
+    display: block;
+    top: 35px;
+  }
+
+  .main {
+    margin-top: 0;
+    background: top right no-repeat url(img/frame1024.svg);
+    background-size: 55%;
+  }
+
+  .main__title {
+    margin: 0;
+    font: 50px/66px pt_serifbold, arial, helvetica, sans-serif;
+  }
+
+  .main__subtitle {
+    margin-top: 40px;
+    padding: 0;
+    font: 17px/20px roboto, arial, helvetica, sans-serif;
+  }
+
+  .main__button {
+    margin-top: 41px;
+    padding: 13px 16px 15px 23px;
+    font: 14px/16px roboto, arial, helvetica, sans-serif;
+  }
+}
+
+@media (max-width: 800px) {
+  .wrapper {
+    margin: auto;
+    width: 80%;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .logo__name {
+    opacity: 0;
+  }
+
+  .header {
+    display: flex;
+    justify-content: space-between;
+    margin: 23px 0 0 0;
+  }
+
+  .main {
+    text-align: center;
+    margin-top: 32px;
+    padding: 80% 0 0;
+    background-size: 80%;
+    background-position: top center;
+  }
+
+  .main__title {
+    margin-top: 5px;
+    font: 30px/40px pt_serifbold, arial, helvetica, sans-serif;
+  }
+
+  .main__subtitle {
+    margin-top: 30px;
+    font: 16px/19px roboto, arial, helvetica, sans-serif;
+  }
+
+  .main__button {
+    margin-top: 28px;
+    padding: 13px 16px 15px 23px;
+    font: 14px/16px roboto, arial, helvetica, sans-serif;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/HTML_SNN/pullRequest?prid=6320070)